### ISSUE TITLE
Tensorboard webpage does not show anything

### ISSUE THREAD

**Issue by sandra1714** [tensorflow/tensorboard#681]
It showed the loss functions last week but this week it is not. It shows a very small graph at the upper right corner. I used the command: python tensorboard/tensorboard.py --logdir models/, where models contains files in train/.

Any ideas??
![screen shot 2017-10-26 at 5 28 01 pm](https://user-images.githubusercontent.com/7615305/32082840-321ed374-ba73-11e7-942f-22eb91d5fb05.png)

**Comment by chihuahua** [comment]
This is because a certain node repeats a lot, resulting in a thin sliver of a graph atop the dashboard. If you zoom into the top of the graph, you will see the graph more clearly.

I am still trying to figure out a good solution to this ...

**Comment by jart** [comment]
I'm closing this one out as a duplicate of #403. Long chains like this have been a known issue in the graph viewer for a while. It's a challenging problem to solve and we welcome any feedback from the community.
